packages/components/src/components/code-docs/code-docs.scss

```
@@ -0,0 +1,57 @@
+/* Add styles for component here */
+
+@use "@db-ui/foundations/build/scss/helpers/shadow-dom" as *;
+@use "@db-ui/foundations/build/scss/variables" as *;
+@use "@db-ui/foundations/build/scss/variables.global" as *;
+@use "@db-ui/foundations/build/scss/color-placeholder" as *;
+@use "@db-ui/foundations/build/scss/tonality" as *;
+
+.db-code-docs {
+	@extend %shadow-dom-extend;
+
+	padding: $db-spacing-fixed-md;
+	padding-block-end: calc($db-spacing-fixed-md + $db-sizing-sm);
+	position: relative;
+
+	.code-button {
+		position: absolute;
+		bottom: 0;
+		right: 0;
+	}
+
+	.no-code {
+		margin: auto;
+	}
+
+	.code {
+		@extend %db-bg-secondary;
+		position: absolute;
+		bottom: 0;
+		right: 0;
+		left: 0;
+		top: 0;
+		border-radius: 8px;
+		display: flex;
```

```suggestion
		display: flex;
		justify-content: space-around;
```

having a better vertical rhythm especially on [long contents](https://db-ui.github.io/mono/review/feat-react-show-code/react-showcase/index.html#/alert?tonality=regular&color=neutral-0&page=variant) might be beneficial.